secrets.py

```
@@ -0,0 +1,3 @@
+db_user = "brent"
+db_password = "StrongPassword!@#"
+api_key = "agsd6t7a6stdyg8as6d876&^%&"
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)